python/py-contextvars/Portfile

```diff
@@ -0,0 +1,43 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           python 1.0
+
+name                py-contextvars
+version             2.4
+categories-append   devel
+platforms           darwin
+license             Apache-2
+
+python.versions     36
+
+maintainers         @jandemter openmaintainer
```

Comment:
please also add an e-mail address (see the [guide](https://guide.macports.org/#development.creating-portfile)), so something like:
```{example.org:jandemter} @jandemter} openmaintainer```